assets/charts/control-plane/packet-ccm/templates/loadbalancer.yaml

```diff
@@ -0,0 +1,303 @@
+apiVersion: v1
```

Comment:
I don't think we need this and according to the README.md of `packet-ccm`, this is optional?

Reply:
This is now removed. We use MetalLB component instead.